libraries/script-engine/src/DataViewPrototype.h

```diff
@@ -65,4 +68,5 @@ public slots:
     bool realOffset(qint32& offset, size_t size) const;
 };
 
-#endif // hifi_DataViewPrototype_h
\ No newline at end of file
+#endif // hifi_DataViewPrototype_h
```

Comment:
A couple of weird characters at the end of this line that would be good to get rid of.

Reply:
Yah I kinda noticed that browsing as well, couldn't really find a good commit to add that prune to... but this seems like a good place